pandas/core/generic.py

```diff
@@ -6913,7 +6916,7 @@ def interpolate(
 
         See Also
         --------
-        fillna : Fill missing values using different methods.
+        fillna : Fill missing values using a set value or existing values.
```

Comment:
-1 on changing this
`fillna` options are more than just how many values you provide. It's also `bfill`, `ffill`, `pad` etc. so I think this should be left as is

Reply:
Do you have suggestions on making the distinction between `fillna` and `interpolate` besides vaguely saying it uses "different methods"?

My choice of words with "existing values" is because the `method` parameter is described as

> pad / ffill: propagate last valid observation forward to next valid backfill / bfill: use next valid observation to fill gap.

My interpretation/translation of "valid observations" was into "existing values". Maybe instead of "existing values", I could replace it with "surrounding valid observations".

If this is bikeshedding, I'm happy to change it back to "different methods". Just trying to be precise and informative :smile:

Reply:
I'd keep the original wording esp as on this page fillna is a clickable link:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.interpolate.html****